### ISSUE TITLE
[Feature request] GT, AD, DP etc columns

### ISSUE THREAD

**Issue by danielnavarrogomez** [Steven-N-Hart/vcf-miner#9]
Will be really nice and useful for my use case to have the GT and other information available on a columns by sample, as it is right now I see which samples have un specific SNP but I don't know if they are Hom or Het, DoC and other important information.

**Comment by Steven-N-Hart** [comment]
Good point.  In the interim, you can apply groupings based on Zygosity.
Add Filter ->Custom -> Variants In Group (after you create one) -> Genotype -> Run Filter

**Comment by puva** [comment]
+1 for this feature request. Currently I add a tag in the INFO column with GT_sampleName=HomRef/Het/HomAlt

**Comment by Steven-N-Hart** [comment]
Great news.  Our Team has been funded to add in these new features.  Stay tuned!
